CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+
+namespace CanHazFunny;
+
+public class Jester
+{
+    private JokeService? _Service;
+    public JokeService Service 

```

Comment:
```suggestion
    private IJokeService? _Service;
    public IJokeService Service 

```
This should be your interface